pkg/rancher-desktop/main/commandServer/httpCommandServer.ts

```diff
@@ -630,6 +638,28 @@ export class HttpCommandServer {
       }
     }
   }
+
+  protected getBackendState(_: express.Request, response: express.Response, context: commandContext): Promise<void> {
+    const backendState = this.commandWorker.getBackendState();
+
+    console.debug('GET backend_state: succeeded 200');
+    response.status(200).json(backendState);
+
+    return Promise.resolve();
+  }
+
+  protected async setBackendState(request: express.Request, response: express.Response, context: commandContext): Promise<void> {
+    const [data] = await serverHelper.getRequestBody(request, MAX_REQUEST_BODY_LENGTH);
+    const state = JSON.parse(data);
+
+    console.debug('PUT backend_state: succeeded 202');
+    response.status(202).type('txt').send('Received backend state.');
+    setImmediate(() => {
+      this.commandWorker.setBackendState(state);
```

Comment:
Good enough; it'll eventually blow up anyway.

(Note that random users can end up calling this too, though; our support is terrible, but it's _doable_.)

Reply:
```
$ rdctld api /v1/backend_state -X PUT -b '{"vmState":"hamster","locked":false}'
Received backend state.
$ echo $?
0
$ cat .../background.log
…

==> background.log <==
2023-09-29T21:32:45.362Z: UnhandledRejectionWarning: Error: invalid desired VM state hamster
    at BackgroundCommandWorker.setBackendState (/Users/ericp/workspace/rancher/desktop/dist/app/background.js:43290:23)
    at Immediate.<anonymous> (/Users/ericp/workspace/rancher/desktop/dist/app/background.js:28082:32)
    at process.processImmediate (node:internal/timers:466:21)

```
I think you need to check `state.vmState` before calling `setBackendState` because right now the exception in `background.ts:setBackendState` gets thrown to the wind because the call is in a `setImmediate`.